symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/UserService.java

```diff
@@ -730,13 +730,33 @@ public V1AuditTrailInitiatorList listAuditTrail(@Nonnull Long startTimestamp, Lo
   }
 
   /**
-   * Suspends or re-activates a user account.
+   * Suspends a user account.
+   * Calling this endpoint requires a service account with the User Provisioning role.
    *
-   * @param userId  user id
-   * @param userSuspension action to perform on the user
+   * @param userId  user id to suspend
+   * @param reason  reason why the user has to be suspended
+   * @param until   timestamp in milliseconds
    * @see <a href="https://developers.symphony.com/restapi/reference#suspend-user-v1">Suspend User Account v1</a>
    */
-  public void suspendUser(@Nonnull Long userId, @Nonnull UserSuspension userSuspension) {
+  public void suspendUser(@Nonnull Long userId, @Nonnull String reason, @Nonnull Long until) {
```

Comment:
That could be more developer friendly to pass a `LocalDateTime` instead of a `Long` value here. WDYT?

Reply:
I would maybe not check the validity as it is server side logic

Reply:
maybe just an Instant instead of Long or LocalDateTime?

Reply:
I checked in the other endpoints we are using Instant for example for the "since" query parameter. IMO I would go for that in here as well.

Reply:
Done https://github.com/finos/symphony-bdk-java/pull/601/commits/f172a527fd55cf4b29bc52224cc3f757026e90a1